scripts/prepare-release.sh

```diff
@@ -172,6 +172,55 @@ check_csv_replaces_field() {
   fi
 }
 
+# Sets the related images in the CSV for RHOAM
+set_related_images() {
+  echo "Adding related images to the CSV"
+  position=0
+  # Get supported components
+  for (( i=0; i<$(yq r -j ./products/products.yaml | jq -r '.products' | jq length); i++))
+  do
+    product_dir=$(yq r ./products/products.yaml "products[$i].manifestsDir")
+    if [[ $(yq r ./products/products.yaml "products[$i].installType") == *"rhoam"* && $(yq r ./products/products.yaml "products[$i].quayScan") == true ]]; then
+      # Read component version
+      component_version=$(grep currentCSV manifests/$product_dir/*.package.yaml | awk -F v '{print $2}')
+
+      # Read component name
+      component_name=$(yq r -j ./manifests/$product_dir/${component_version}/*.clusterserviceversion.yaml | jq '.metadata.name' |  tr -d '"')
+      
+      # Read image from the component version but only select quay.io or redhat.registry
+      component_image=$(yq r -j ./manifests/$product_dir/${component_version}/*.clusterserviceversion.yaml | jq '.spec.install.spec.deployments[0].spec.template.spec.containers' | jq '.[] | select((.image|test("quay.")) or (.image|test("registry.redhat"))) | .image' |  tr -d '"')
+
+      # Push image to relatedImages in RHOAM CSV
```

Comment:
Is this meant to be `containerImages` instead of `relatedImages`?

```suggestion
      # Push image to containerImages in RHOAM CSV
```